_inc/connect-button.js

```diff
@@ -25,11 +26,9 @@ jQuery( document ).ready( function( $ ) {
 					_wpnonce: jpConnect.apiNonce,
 				},
 				error: function( error ) {
-					console.warn( error );
+					console.warn( 'Connection failed. Falling back to the regular flow', error );
 					jetpackConnectButton.isRegistering = false;
-					$( '.jp-connect-button' )
-						.text( jpConnect.buttonTextDefault )
```

Comment:
Seems like this is the last usage of `jpConnect.buttonTextDefault` so we could totally remove that one from `jpConnect`.

Reply:
Thanks for catching this up. I've addressed it in c54bb0b.